apps/dashboard/src/data/actions.js

```diff
@@ -1,10 +1,16 @@
 /**
  * Internal dependencies
  */
-import { ASYNC_ROUTINE_DISPATCH } from './action-types';
+import { ASYNC_ROUTINE_DISPATCH, FINISH_RESOLUTION } from './action-types';
 
 export const dispatchAsync = ( apply, args = [] ) => ( {
 	type: ASYNC_ROUTINE_DISPATCH,
 	apply,
 	args,
 } );
+
+export const finishResolution = ( selectorName, args ) => ( {
```

Comment:
It might be worth adding a docblock comment here with a link to the original action or something - basically explaining what this is and why it's here.